docs/current_docs/quickstart/basics/basics.mdx

```diff
@@ -0,0 +1,288 @@
+---
+slug: /quickstart/basics
+title: "Basics"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Basics
+
+Welcome to Dagger, a general-purpose composition engine for containerized workflows.
+
+Dagger is a modular, composable platform designed to replace complex systems glued together with artisanal scripts - for example, complex integration testing environments, data processing pipelines, and AI agent workflows. It is open source and works with any compute platform or technology stack, automatically optimizing for speed and cost.
+
+## Creating containers
+
+Dagger works by expressing workflows as combinations of functions from the Dagger API, which exposes a range of tools for working with containers, files, directories, network services, secrets. You call the Dagger API from shell (Dagger Shell) or code (custom Dagger Functions written in a programming language).
+
+Dagger Shell is an interactive client for the Dagger API, giving you typed objects, built-in documentation, and access to a cross-language ecosystem of reusable modules. You launch it simply by typing `dagger`. Here's an example of using it to build and return an Alpine container:
+
+```shell
+container |
+  from alpine
+```
+
+[FIXME recording]
+
+## Adding files and directories
```

Comment:
We should show `.help` before this so that all the chained functions don't feel as mysterious